docs/tasks/administer-cluster/kube-router-network-policy.md

```diff
@@ -0,0 +1,25 @@
+---
+approvers:
+- murali-reddy
+title: Using Kube-router for NetworkPolicy
+---
+
+{% capture overview %}
+This page shows how to use [Kube-router](https://github.com/cloudnativelabs/kube-router) for NetworkPolicy.
+{% endcapture %}
+
+{% capture prerequisites %}
+* Install Kube-router add-on for Kubernetes.
```

Comment:
Is this documentation meant to address installation or usage? 

If installation, then the prerequisites should be something like:
* Have a Kubernetes environment running.
* Be familiar with [Kops](https://github.com/kubernetes/kops), [Bootkube](https://github.com/kubernetes-incubator/bootkube), or [Kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/).
* Make sure you satisfy the [requirements](https://github.com/cloudnativelabs/kube-router/tree/master/Documentation#requirements) for Kube-router.

Otherwise, if this is meant to address usage, then I would move *Installing Kube-router addon* into the *prerequisites* section and add sections on configuration and usage in the  *steps* section.

Reply:
Yes it meant for installation. Usage really does not vary between the network policy providers listed. Kubernetes network policy API is how one defines and uses network policies. Its only installation detalation varying.